python/cudf/cudf/tests/test_numerical.py

```diff
@@ -42,6 +46,27 @@ def test_can_cast_safely_mixed_kind():
     data = Series([1.0, 2.0, 1.0 * (2 ** 31)], dtype="float32")._column
     assert not data.can_cast_safely(to_dtype)
 
+@pytest.mark.xfail
+def test_to_pandas_nullable_integer():
+    gsr_not_null = Series([1,2,3])
+    gsr_has_null = Series([1,2,None])
+
+    psr_not_null = pd.Series([1,2,3], dtype='int64')
+    psr_has_null = pd.Series([1,2,None], dtype='Int64')
+
+    assert_eq(gsr_not_null.to_pandas(), psr_not_null)
+    assert_eq(gsr_has_null.to_pandas(), psr_has_null)
+
+@pytest.mark.xfail
+def test_to_pandas_nullable_bool():
+    gsr_not_null = Series([True, False, True])
+    gsr_has_null = Series([True, False, None])
+
+    psr_not_null = pd.Series([True, False, True], dtype='bool')
+    psr_has_null = pd.Series([True, False, None], dtype='boolean')
+
+    assert_eq(gsr_not_null.to_pandas(), psr_not_null)
+    assert_eq(gsr_has_null.to_pandas(), psr_has_null)
```

Comment:
We should open an issue for discussion about whether we should default to using these new nullable types when calling `to_pandas` or if we should continue using the non-nullable types.

Or we can just always go through Arrow and let them dictate our behavior 😄

Reply:
Hoping to collect some thoughts on this here. https://github.com/rapidsai/cudf/issues/4823 My initial attempt to support this compatibility as part of this PR led to a lot of issues, I thought it was best to decouple that support from this PR.